src/components/Login/index.md

```diff
@@ -2,50 +2,49 @@
 title:
   en-US: Login
   zh-CN: Login
-subtitle: 登录
+subtitle: Login
```

Comment:
You should create both `index.en-US.md` and `index.zh-CN.md`

Reply:
Thanks. Totally missed that. I've updated it.